castReceiver/index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bitmovin Cast Receiver Demo</title>
+    <meta charset="UTF-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400" rel="stylesheet">
+    <link rel="icon" type="image/png" href="../images/bit-fav.png">
+    <!-- Bitmovin Player -->
+    <script src="//cdn.bitmovin.com/player/web/8.5.0/bitmovinplayer.js"></script>
```

Comment:
As on the receiver we're using the latest version 8 player version we probably want to use it here too

Reply:
AFAIK we have some issues if the versions are not matching. For example I tried player `8.5.0` with the `8.0.0/bitmovinplayer-remotereceiver.js` and nothing worked (also no error message was shown). And we don't have a `8.latest` link to the `bitmovinplayer-remotereceiver.js ` yet so I explicitly decided to use specific versions here. But sure we could use `8.6.0`